pipeline.yaml

```diff
@@ -61,6 +67,13 @@ steps:
     # Indicates whether or not a model that fails to meet performance thresholds should still
     # be registered to the MLflow Model Registry
     allow_non_validated_model: true
+  predict:
+    # Specify the model URI to use for batch scoring
+    model_uri: "models/model.pkl"
```

Comment:
Can we comment out this line? By default the scoring graph should look for a model produced from the train step. You can add a comment to indicate that users can however override this behavior by providing a model URI supported by MLflow: https://www.mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#mlflow.pyfunc.load_model

Reply:
Yep.  Sounds good.  Jerry and I will fix up the tests in the feature branch to work with this.

Reply:
Commented out and updated the comment.